argocd-diff/action.yml

```
@@ -0,0 +1,35 @@
+name: "ArgoCD Diff"
+description: "argocd diff"
+inputs:
+  argocd-app:
+    description: "ArgoCD app name of cluster"
```

Minor nit: maybe change this to be a bit more descriptive for generic use
```suggestion
    description: "Name of App of apps used in manifest"
```